cpp/README.rst

```diff
@@ -32,6 +32,21 @@ Building and running the code examples
 Additional information geared specifically to understanding the example applications is at
 `AWS SDK for C++ Code Examples <https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/programming-services.html>`_.
 
+
+Docker image (Beta)
+===================
+
+In the near future, this example code will be available in a container image
+hosted on [Amazon Elastic Container Registry (ECR)](https://docs.aws.amazon.com/AmazonECR/latest/userguide/what-is-ecr.html). This image will be pre-loaded
+with all C++ examples with dependencies pre-resolved, allowing you to explore
+these examples in an isolated environment.
+
+As of January 2023, the [SDK for C++ image](https://gallery.ecr.aws/aws-docs-sdk-examples/cpp) is available on ECR Public but is still
+undergoing active development. Refer to
+[this GitHub issue](https://github.com/awsdocs/aws-doc-sdk-examples/issues/4133)
+for more information.

```

Comment:
Here, and in the other READMEs with this text, recommend the following last sentence update: 

For more information, see [this GitHub issue](https://github.com/awsdocs/aws-doc-sdk-examples/issues/4133).